### ISSUE TITLE
Medalla Launchpad interface to GitHub

### ISSUE THREAD

**Issue by raunchy-mx** [ethereum/eth2.0-deposit#163]
I am trying to use the Medalla Launchpad to deposit goETH as a step to becoming a validator for ETH2. The Launchpad sends me to page with BICORNIS SPRINT as a main heading, but the packages are shown as prerelease an I can’t get further

**Comment by CarlBeek** [comment]
You're almost there @raunchy-mx. Bicornis Sprint is the name of the latest launchpad release. The packages you found are pre-release because this tool hasn't been audited yet,  and mainnet is yet to be launched. Downloading and running the packages you found should just work.

**Comment by raunchy-mx** [comment]
Hi Carl:
Thank you for your response. Perhaps because I'm not familiar with github,
I don't understand how the package (
eth2deposit-cli-v0.2.1-darwin-amd64.tar.gz) can be downloaded since it
doesn't show as an active link.  Do I do something like:
$HOME/prysm/prysm.sh --install -
https://github.com/ethereum/eth2.0-deposit-cli/releases/
eth2deposit-cli-v0.2.1-darwin-amd64.tar.gz

Puzzled,
Bob @raunchy-mx

**Comment by mccabesville** [comment]
I'm having exactly the same issues at this point.
Not easy to locate the deposit code. When I eventually found and ran the deposits asset, I got what should be a 'successful message'; however, when I try to drag and drop the file I get the error message that the json file I've used isn't for the medalla testnet. I got the asset from the Bicornis Sprint section - is this correct?

**Comment by raunchy-mx** [comment]
Hi, mccabesville:

I hope you can help, because you “eventually found and ran the deposits
asset“ and I haven’t been able to get that far. Any help you can provide
would be very much appreciated.

**Comment by mccabesville** [comment]
Hi,

Have a look here: https://github.com/ethereum/eth2.0-deposit-cli/releases/

Under the Bicornis Sprint section there is a dropdown called assets. Chose the Mac, linux or windows option and run that.

I got the successful message but the json file seems to be incorrect.

Sorry I can't be more help.

**Comment by CarlBeek** [comment]
@mccabesville, thanks for trying to help @raunchy-mx out here. Your instructions to them should get them to the download. @raunchy-mx, if you still can't find it I recommend trying [this link](https://github.com/ethereum/eth2.0-deposit-cli/releases/tag/v0.2.1).

@mccabesville, did you choose the "medalla" option when asked witch network you were using? If you just hit enter when the prompt asked about it, you'd end up with keys which can only be used on mainnet.


It should look like this:
```text
Please choose how many validators you wish to run: 1
Please choose your mnemonic language (czech, chinese_traditional, chinese_simplified, english, spanish, italian, korean) [english]: english
Please choose the (mainnet or testnet) network/chain name (mainnet, witti, altona, medalla) [mainnet]: medalla
```

Note the last line where `medalla` is typed. Hopefully this helps.

**Comment by raunchy-mx** [comment]
Yeah, many thanks to @mccabesville for pointing out the “assets” link that
was hidden below a line, which to me implied next subject. I did get hung
up at the connect window when nothing happened after pushing the Metamask
button. After two days, I went through the Launchpad again and the response
was immediate.

Still trying to iron out some details like keystone-data  path and
beaconchaindata path... quite a learning experience for an 80 year old
geezer!
